examples/truffle/test/erc20.spec.js

```diff
@@ -211,8 +211,8 @@ contract('ERC20', (accounts) => {
     const balance02 = await token.balanceOf.call(accounts[ 0 ])
     assert.strictEqual(balance02.toNumber(), 9980)
   })
-
-  /* eslint-disable no-underscore-dangle */
+  
+  // eslint-disable-next-line
```

Comment:
We shouldn't disable ALL checks. I think this is what you want: 
`// eslint-disable-next-line no-underscore-dangle`

Reply:
Does this check even have any impact on the following line?

Reply:
It disables all rules for that line:

> To disable all rules on a specific line, use a line or block comment in one of the following formats:
> 
>...
> 
> // eslint-disable-next-line
> alert('foo');

Source: [Rules](https://eslint.org/docs/user-guide/configuring/rules#disabling-rules)

Reply:
Yeah definitely should be more specific - that's my b

Reply:
Wait yeah lemme check to see if there's actually an impact on next line here